src/ShopBundle/objects/db/TShopOrder.class.php

```diff
@@ -540,6 +543,32 @@ protected function SaveOrderNotificationDataToOrder($bOrderSend, $oMail, $sSendT
         $this->AllowEditByAll(false);
     }
 
+    protected function SetSendOrderNotificationState(bool $orderSend): void
+    {
+        if (true === TGlobal::IsCMSMode()) {
```

Comment:
When or how would this be called in the backend?

Is an early return then the right behavior?